backend/app/Http/Controllers/ProfileController.php

```diff
@@ -80,4 +81,14 @@ public function getUserDetail($user_id)
 
         return response()->build(self::RESPONSE_MESSAGE_SUCCESS, ['cards'=> $cards]);
     }
+
+    /**
+     * Gets all the transactions of the user's purchases.
+     *
+     * @return mixed transactions
+     */
+    public function getMyTransactions()
+    {
+        return response()->build(self::RESPONSE_MESSAGE_SUCCESS, Transaction::where('user_id', auth()->user()->id)->get());
```

Comment:
Eventually we will want to implement this in the same way as getCardDetail, where we can fetch the transaction log for any given `user_id`.

This is fine for now if you find it easier to do this intermittent step, and then later someone will write the new method.

https://github.com/CryptoCardsETH/cryptocards/blob/master/backend/app/Http/Controllers/ProfileController.php#L71